### PR TITLE
Old Link in migrating_from_maven.adoc

### PR DESCRIPTION
**Pull request by hythloda**
Fixes #?

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [x] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [x] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [x] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/migration/migrating_from_maven.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@
 [[migrating_from_maven]]
 = Migrating Builds From Apache Maven
 
-TIP: Suffering from slow Maven builds? https://gradle.com/training/build-cache-deep-dive/?bid=docs-migrating-maven[Register here] for our Build Cache training session to learn how Gradle Enterprise can speed up Maven builds by up to 90%.
+TIP: Suffering from slow Maven builds? https://gradle.com/training[Register here] for our Build Cache training session to learn how Gradle Enterprise can speed up Maven builds by up to 90%.
```
Comment (big-guy) [inline subprojects/docs/src/docs/userguide/migration/migrating_from_maven.adoc:18]: is this supposed to be https://gradle.com/training/#build-cache-deep-dive too?

File: subprojects/docs/src/docs/userguide/running-builds/performance.adoc [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-Builds run many times a day, so even small waiting periods add up.
-The same is true for Continuous Integration (CI) builds: the less time they take, the faster you can react to new issues
-and the more often you can experiment.
+TIP: Want faster Gradle Builds? https://gradle.com/training/#build-cache-deep-dive[Register here] for our Build Cache training session to learn how Gradle Enterprise can speed up builds by up to 90%.
+
+Build performance is critical to your productivity.
+The longer the build takes to complete, the more likely you'll be taken out of your development flow.
+On top of that, since you run the build many times a day, even small periods of waiting can add up to significant disruption.
+The same is true for builds run on CI: the faster they are, the faster you can react to new issues and the more capacity you will have to do innovative experiments.
+
```
Comment (nathan-contino) [inline subprojects/docs/src/docs/userguide/running-builds/performance.adoc:10]: It looks like you've inadvertently brought an old paragraph of prose into the updated performance page. This tip now exists 12 lines below this new tip, and this paragraph has been replaced by a different paragraph. Could you restore this to the recently updated state and instead update the tip on line 16?

**Review by nathan-contino — CHANGES_REQUESTED** [review]
Looks good apart from a regression back to an older version of the performance page intro. Once you've fixed that, happy to LGTM.

File: subprojects/docs/src/docs/userguide/running-builds/performance.adoc (hunk 2 above)
Comment (hythloda) [reply subprojects/docs/src/docs/userguide/running-builds/performance.adoc:10]: thanks that was a merge conflict I resolved the wrong way ;)

**Review by nathan-contino — APPROVED** [review]
LGTM

**Review by big-guy — APPROVED** [review]
@hythloda if you want this to make 7.6, you'll need to cherry-pick this back to the release branch

**Comment by hythloda** [comment]
replaced by 21675

**Pull request closed without merging**